Baikal/PostEffects/ML/denoiser.cpp

```diff
@@ -22,5 +52,150 @@ namespace Baikal
                                                    height,
                                                    input_channels);
         }
+
+        MLDenoiser::MLDenoiser(CLWContext context, Inference::Ptr inference, MLDenoiserInputs inputs)
+                   : m_inference(std::move(inference))
+        {
+            m_context = std::move(std::make_unique<CLWContext>(context));
+            m_primitives = std::move(std::make_unique<CLWParallelPrimitives>(context));
+
+            auto shape = m_inference->GetInputShape();
+            auto width = std::get<0>(shape);
+            auto height = std::get<1>(shape);
+            auto chanels_num = std::get<2>(shape);
+
+            size_t elems_count = sizeof(Tensor::ValueType) * width * height * chanels_num;
+
+            m_device_cache = std::move(std::make_unique<CLWBuffer<char>>(
```

Comment:
Why do we need a unique_ptr here? Why not just CLWBuffer<char>?
std::move is not required in any case here.

Reply:
First, there is no necessary include in header file

Reply:
Second, there is no suitable constructor for CLWBuffer